LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2013 Puppet Labs
+   Copyright 2016 Puppet, Inc.
```

Comment:
Hrmm...Don't think so. The license on Puppet doesn't/didn't have it.  Although I was the one that updated it.